modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/ConfigurationAsmGenerator.java

```diff
@@ -1324,7 +1326,16 @@ private void addNodeConstructMethod(
                     .ret();
         }
     }
-    
+
+    private void makeSureChangePolymorphicTypeIdMethodIsDefined(@Nullable MethodDefinition changePolymorphicTypeIdMtd,
+                                                                Field schemaField) {
+        if (changePolymorphicTypeIdMtd == null) {
+            throw new IllegalStateException("Field " + schemaField.getDeclaringClass().getName() + "."
+                    + schemaField.getName() + " is annotated with @PolymorphicId, but schema extensions are "
```

Comment:
It would be more correct to make it `"@" + PolymorphicId.class.getSimpleName()` so that if the name of the annotation changes, it would change automatically.

Reply:
Done